src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -544,6 +578,12 @@ const PIAFormPage = () => {
       }
       event.preventDefault();
       event.stopPropagation();
+
+      navigate(
+        buildDynamicPath(routes.PIA_INTAKE_EDIT, {
+          id: pia?.id,
+        }),
+      );
```

Comment:
why navigate if the form is invalid?

Reply:
it is in AC. we need to route the user to the error place

Reply:
Don't we need to scroll to the error? Why navigate? Line 576 will do it for you.

Reply:
this is for view mode redirect 
<img width="957" alt="Screenshot 2023-03-28 at 8 43 53 AM" src="https://user-images.githubusercontent.com/57013495/228293709-2bf926b8-49fd-4203-bc36-5498d2594565.png">

Reply:
let us clarify with Jordan about this. This seems a bit convoluted.